scripts/scil_filter_streamlines_by_length.py

```diff
@@ -72,6 +79,12 @@ def main():
 
     save_tractogram(new_sft, args.out_tractogram)
 
+    tc_bf = len(sft.streamlines)
+    tc_af = len(new_streamlines)
+    print(json.dumps({'tract_count_before_filtering': int(tc_bf),
```

Comment:
You added an argument (`--json`) to know if the user wants the values printed or not, but here you don't check if it is set.

Reply:
fix